pages/cookies.md

```diff
@@ -43,4 +44,12 @@ This helps us identify how you use this site so we can make it better.
 This prevents too many requests being made to the Google Analytics service.
 *10 minutes*
 
-Google isn’t allowed to use or share our analytics data.
\ No newline at end of file
+Google isn’t allowed to use or share our analytics data.
+
+### Play video content (Vimeo)
+Vimeo’s embeddable video player uses essential cookies.
+
+The following cookies are used:
+
+* **player 1 year**
```

Comment:
Can we follow the same format for the Google Analytics cookies above?

Reply:
This is what they look like in the deploy preview 
![image](https://user-images.githubusercontent.com/632390/74439047-5d2a3f00-4e63-11ea-97a2-5b406a6563cb.png)

Reply:
fixed 👍